src/core/manager.c

```diff
@@ -1740,6 +1740,9 @@ int manager_add_job(
         if (mode == JOB_ISOLATE && !unit->allow_isolate)
                 return sd_bus_error_setf(error, BUS_ERROR_NO_ISOLATION, "Operation refused, unit may not be isolated.");
 
+        if (mode == JOB_TRIGGERING && type != JOB_STOP)
+                return sd_bus_error_setf(error, SD_BUS_ERROR_INVALID_ARGS, "Triggering is only valid for stop.");
```

Comment:
Maybe `--mode=triggering is only ...`. This error message might be unclear without context.